### ISSUE TITLE
byHashtag There's a lot less data

### ISSUE THREAD

**Issue by wangzsPalpitate** [davidteather/TikTok-Api#637]
Before we could get thousands of them, now we can only get dozens of them. Is this an anomaly

**Comment by dylancaponi** [comment]
Have you tried changing verifyFp and the IP you're requesting from?

**Comment by wangzsPalpitate** [comment]
> 您是否尝试过更改 verifyFp 和您请求的 IP？

I've tried, but it doesn't work

**Comment by wangzsPalpitate** [comment]
I took a closer look, and as I turned the page, the CAPTCHA appeared

**Comment by jDan735** [comment]
Got the same problem with the `item_list` api method. It affects methods `by_trending`, `user_posts`, `user_page`, `user_liked`, `by_sound`, `by_hashtag`, `get_recommended_tiktoks_by_video_id` (`by_sound`, `by_hashtag` definitely require captcha).

**Comment by davidteather** [comment]
There might've been a few issues with this that might've been fixed in a new release with some new naming conventions. Feel free to read about how to convert and upgrade [V4.0.0](https://github.com/davidteather/TikTok-Api/releases/tag/V4.0.0)

**Comment by wangzsPalpitate** [comment]
> 可能存在一些与此有关的问题，这些问题可能已在具有一些新命名约定的新版本中修复。随意阅读如何转换和升级[V4.0.0](https://github.com/davidteather/TikTok-Api/releases/tag/V4.0.0)

Thank you very much for providing such a powerful API. There are a few more BY_hashtags in 4.0 than in the previous version. Is there any way to get more by_hashtag data

**Comment by davidteather** [comment]
I was messing around with it, seems like they might be restricting it on their test servers to make sure that `use_test_servers=False` not sure any fixes beyond that.

**Comment by Incher25** [comment]
> Got the same problem with the `item_list` api method. It affects methods `by_trending`, `user_posts`, `user_page`, `user_liked`, `by_sound`, `by_hashtag` (`by_sound`, `by_hashtag` definitely require captcha).

I got this problem too - getting tiktok by id works just fine, but "batch"-retrieving methods like `user_posts`, `user_page`, `by_username` fail by captcha.

**Comment by dylancaponi** [comment]
> use_test_servers=False

`use_test_endpoints=False` ?

**Comment by Incher25** [comment]
> `use_test_endpoints=False` ?
It makes no difference - in both cases I get the same captcha error.

**Comment by davidteather** [comment]
Looks to be fixed with V4.1.0
```
>>> h = api.by_hashtag("funny")
>>> len(h)
30
>>> h = api.by_hashtag("funny", count=500)
>>> len(h)
500  
```
